Assets/Scenes/SampleScene.unity

```diff
@@ -9957,8 +9957,10 @@ MonoBehaviour:
     obj: {fileID: 21300000, guid: a1dd56a05c598b34eb83acfc85921c84, type: 3}
   - ID: ability_indicator_absorb_field
     obj: {fileID: 21300000, guid: b7672a0b658b7cf418d34b7c8cbe6d51, type: 3}
-  fieldID: groundcarriercore
-  newObject: {fileID: 21300000, guid: 7abd63c2e847e994bb0195f2d53a066f, type: 3}
```

Comment:
What was that script for?

Reply:
> What was that script for? Just wondering (and i didn't mean to be a reviewer)

i'm not sure, this was the default entry in the resource manager and it didn't seem to actually replace the ground core so I assumed it was just an accident